spec/models/on/empty_array_spec.rb

```diff
@@ -21,6 +21,15 @@ module On
       end
     end
 
+    describe ".paper_trail.update_columns" do
+      it "creates a version record" do
+        widget = Widget.create
+        assert_equal 1, widget.versions.length
+        widget.paper_trail.update_columns(name: "Bugle")
+        assert_equal 2, widget.versions.length
```

Comment:
Just FYI, there is a nice way to "expect a change" in RSpec. You don't have to use it, what you have is fine, just thought I'd share.

```
expect {
  widget.paper_trail.update_columns(name: "Bugle")
}.to(change { widget.versions.length }).from(1).to(2)
```